specification/loadtestservice/resource-manager/Microsoft.LoadTestService/stable/2022-12-01/loadtestservice.json

```diff
@@ -0,0 +1,963 @@
+          "type": "object",
+          "description": "All identity configuration for Customer-managed key settings defining which identity should be used to auth to Key Vault.",
+          "properties": {
+            "type": {
+              "type": "string",
+              "enum": [
+                "SystemAssigned",
+                "UserAssigned"
+              ],
+              "x-ms-enum": {
+                "name": "type",
+                "modelAsString": true
+              },
+              "description": "Managed identity type to use for accessing encryption key Url"
+            },
+            "resourceId": {
+              "type": "string",
+              "description": "user assigned identity to use for accessing key encryption key Url. Ex: /subscriptions/fa5fc227-a624-475e-b696-cdd604c735bc/resourceGroups/<resource group>/providers/Microsoft.ManagedIdentity/userAssignedIdentities/myId"
+            }
+          }
+        },
+        "keyUrl": {
+          "type": "string",
+          "description": "key encryption key Url, versioned. Ex: https://contosovault.vault.azure.net/keys/contosokek/562a4bb76b524a1493a6afe8e536ee78 or https://contosovault.vault.azure.net/keys/contosokek."
+        }
+      }
+    },
+    "ResourceState": {
+      "description": "Resources provisioning states.",
+      "enum": [
+        "Succeeded",
+        "Failed",
+        "Canceled",
+        "Deleted"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "ResourceState",
+        "modelAsString": true
+      }
+    },
+    "QuotaResourceList": {
+      "description": "A list of Quota Bucket details. It contains an URL link to get the next set of results.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/QuotaResource"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ],
+          "description": "List of Quota Bucket details by the loadtestservice resource provider",
+          "readOnly": true
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to get the next set of Quota Bucket details results (if there are any).",
+          "readOnly": true
+        }
+      }
+    },
+    "QuotaResource": {
+      "type": "object",
+      "description": "Quota Bucket details",
+      "properties": {
+        "properties": {
+          "description": "Quota resource properties",
+          "$ref": "#/definitions/QuotaResourceProperties",
+          "x-ms-client-flatten": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "QuotaResourceProperties": {
+      "description": "Quota resource properties.",
+      "type": "object",
+      "properties": {
+        "limit": {
+          "description": "Quota Limit of the quota bucket.",
```

Comment:
>Quota Limit of the quota bucket [](http://example.com/codeflow?start=26&length=31)

Units?

Reply:
This is a temp API as RPaaS doesn't support the ARM Usages API which contains this - https://stackoverflow.microsoft.com/questions/265563
So, for time being we're following the structure for integration with QMS team -
https://eng.ms/docs/cloud-ai-platform/ahsi-organization/ahsi/cscp-teams/cscp-quotamanagement/azure-quota-services/policy-qms/resource-provider/ref-get-usages-and-quota @mentat9

Reply:
Just suggesting a description that explains it better. These are your user docs, not blocking ARM signoff.

Reply:
Done